index.bs

```diff
@@ -5943,6 +6012,11 @@ Each stored [=virtual authenticator=] has the following properties:
     [=User Verification=] will always succeed. If set to [FALSE], it will fail.
 
     Note: This property has no effect if |hasUserVerification| is set to [FALSE].
+: |extensions|
+:: A string array containing the [=extension identifiers=] supported by the [=Virtual Authenticator=].
+
+[=Virtual authenticators=] MUST support all [=authenticator extensions=] present in its |extensions| array.
+They MUST NOT support any [=authenticator extension=] not present in its |extensions| array.
```

Comment:
There's a mix of plural and singular here. I would suggest:

```suggestion
A [=virtual authenticators=] MUST support all [=authenticator extensions=] present in its |extensions| array.
It MUST NOT support any [=authenticator extension=] not present in its |extensions| array.
```

Reply:
Might also be appropriate to indent this as a second paragraph of the `|extensions|` definition? I'm not sure.